en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -303,6 +303,53 @@ class BreadAPIController extends Controller
 In Silverstripe CMS versions prior to 4.6, an empty key (`''`) must be used in place of the `'/'` key. When specifying an HTTP method, the empty string must be separated from the method (e.g. `'GET '`). The empty key and slash key are also equivalent in Director rules.
 [/alert]
 
+## Nested RequestHandlers
+
+Nested [`RequestHandler`](api:SilverStripe\Control\RequestHandler) routing is used extensively in the CMS and is used to create URL endpoints without yml configuration.  Nesting is done by returning a [`RequestHandler`](api:SilverStripe\Control\RequestHandler) from an action method on another [`RequestHandler`](api:SilverStripe\Control\RequestHandler), usually a [`Controller`](api:SilverStripe\Control\Controller).
```

Comment:
```suggestion
Nested [`RequestHandler`](api:SilverStripe\Control\RequestHandler) routing is used extensively in the CMS and is used to create URL endpoints without yml configuration.  Nesting is done by returning a `RequestHandler` from an action method on another `RequestHandler`, usually a [`Controller`](api:SilverStripe\Control\Controller).
```
We only need a link the first time we mention it. Too many links will make the text harder to read.

Reply:
Updated